src/mono/mono/metadata/reflection.c

```diff
@@ -3131,7 +3131,7 @@ mono_reflection_call_is_assignable_to (MonoClass *klass, MonoClass *oklass, Mono
 	error_init (error);
 
 	if (method == NULL) {
-		method = mono_class_get_method_from_name_checked (mono_class_get_type_builder_class (), "IsAssignableTo", 1, 0, error);
+		method = mono_class_get_method_from_name_checked (mono_class_get_type_builder_class (), "IsAssignableToInternal", 1, 0, error);
 		mono_error_assert_ok (error);
```

Comment:
This change needs to be inside an #ifdef ENABLE_NETCORE since the old mono/mono still uses the old method name.

Reply:
Done